### PR TITLE
[OSDEV-1153] Create integration tests for the OpenSearch

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-1153](https://opensupplyhub.atlassian.net/browse/OSDEV-1153) Create integration tests for the OpenSearch

- Create Integration Tests for OpenSearch
- Create Production Locations endpoint tests

[OSDEV-1153]: https://opensupplyhub.atlassian.net/browse/OSDEV-1153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by VadimKovalenkoSNF — COMMENTED** [review]
Asked few questions.

File: docker-compose.tests.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    build:
+      context: ./src/tests
+      dockerfile: Dockerfile
+    environment:
+      - OPENSEARCH_HOST=opensearch-single-node
+      - OPENSEARCH_PORT=9200
+      - OPENSEARCH_INDEX_NAME=production-locations
+      - REACT_HOST=react
+      - REACT_PORT=6543
+      - USER_API_TOKEN=1d18b962d6f976b0b7e8fcf9fcc39b56cf278051
```
[the remark this answers is not in the document]
Comment (roman-stolar) [reply docker-compose.tests.yml:17]: @VadimKovalenkoSNF Yes, this token is created automatically when we fill database with fixtured data.

File: scripts/start_local_dev [hunk 2]
```diff
@@ -18,6 +18,13 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         echo "Building Docker images and creating database structure..."
         ./scripts/update
 
+        # Collect Django static files
```
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply scripts/start_local_dev:21]: Got it.

File: docker-compose.tests.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply docker-compose.tests.yml:17]: Got it, thanks.

**Review by VadimKovalenkoSNF — APPROVED** [review]
Looks ok, thank you for implementing this.

File: scripts/start_cq_dev
Comment (mazursasha1990) [inline scripts/start_cq_dev:1]: Please rename this file to `start_code_quality_dev`, because the `cq` is not immediately clear.

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
In general, looks good to me. 
But I would recommend to rename the file.

File: scripts/start_cq_dev
Comment (roman-stolar) [reply scripts/start_cq_dev:1]: @mazursasha1990 Done.

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
LGTM

**Pull request merged**